src/analyze/State.re

```diff
@@ -429,7 +469,7 @@ let newJbuilderPackage = (~reportDiagnostics, state, rootPath) => {
     buildSystem,
     buildCommand,
     /* TODO check if there's a module called that */
-    opens: fold(libraryName, [], libraryName => [libraryName ++ "__"]),
+    opens: fold(libraryName, [], libraryName => [String.capitalize(libraryName)]),
```

Comment:
Ah, so we need a check here.
e.g. if the library is named `Dogs` and it contains a `Dogs.re`, then the open will be `Dogs__`. Otherwise it will be `Dogs`.
But I already have the TODO indicating this, so I'm willing to defer

Reply:
Ahhhh, that makes much more sense. I had noticed a discrepancy between a couple local projects which had me scratching my head but this explains it. Do you have an idea of how to do this detection, or is that still an open question? I'm ok either with fixing it in this PR or deferring, your call.